OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -156,6 +156,12 @@ public bool FaceTarget(Actor self, Target target)
 			return HasAchievedDesiredFacing;
 		}
 
+		public void StopAimingIfAiming(Actor self)
```

Comment:
I think you can drop the 'IfAiming', in my opinion that should be obvious from the context.